src/layer_normalization.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+import {add, sub, mul, div, pow} from './binary.js';
+import {reduceMean} from './reduce.js';
+import {reshape} from './reshape.js';
+import {Tensor, Scalar} from './lib/tensor.js';
+import {validateLayerNormalizationParams} from './lib/validate-input.js';
+
+/**
+ * Normalize the tensor values of input features using
+ * [layer-Normalization]
```

Comment:
Please update **[layer-Normalization]** with link, and combine two lines into just one line 
```
 * Normalize the tensor values of input features using
 * [layer-Normalization]
```

to
```
 * Normalize the tensor values of input features using [layer-Normalization](https://arxiv.org/abs/1607.06450)
```

Reply:
ok，thanks